.github/workflows/linting.yml

```diff
@@ -93,3 +93,46 @@ jobs:
           command: yarn --no-audit --prefer-offline
       - name: Lint
         run: yarn run tsc:compile
+
+
+  typedoc:
+    name: TypeDoc Generation
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        name: Yarn Cache
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/package.json') }}-with-website
```

Comment:
interesting I think if you just put the yarn-cache from checkout package.json doesn't have the full thing yet so that key won't be right. This seems like a fine workaround given that it's not in the workspace so not in package.json by default.